### PR TITLE
Add XKCD Presence

### PR DESCRIPTION
**Pull request by theusaf**
## What is XKCD?
XKCD is a webcomic of romance, sarcasm, math, and language made by Randall Munroe.
## Example:
**Non-Comic Page**
Browsing XKCD
**Comic Page**
Reading Comic
**Title Text**
Title text on small image

### CODE REVIEW

File: websites/X/XKCD/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    "xkcd.com",
+    "www.xkcd.com"
+  ],
+  "description": {
+    "en": "XKCD is a webcomic of romance, sarcasm, math, and language."
+  },
+  "service": "XKCD",
+  "version": "0.0.1",
+  "logo": "https://cdn.discordapp.com/attachments/710280407777673277/754584949452963882/xkcd_illustration_x4.jpg",
+  "thumbnail": "https://cdn.discordapp.com/attachments/710280407777673277/754552040545583302/Screen_Shot_2020-09-12_at_8.59.55_PM.png",
```
Comment (Bas950) [inline websites/X/XKCD/dist/metadata.json:17]: upload to imgur

File: websites/X/XKCD/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  },
+  "url": [
+    "xkcd.com",
+    "www.xkcd.com"
+  ],
+  "description": {
+    "en": "XKCD is a webcomic of romance, sarcasm, math, and language."
+  },
+  "service": "XKCD",
+  "version": "0.0.1",
```
Comment (Bas950) [inline websites/X/XKCD/dist/metadata.json:15]: version 1.0.0 on release

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/X/XKCD/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    "xkcd.com",
+    "www.xkcd.com"
+  ],
+  "description": {
+    "en": "XKCD is a webcomic of romance, sarcasm, math, and language."
+  },
+  "service": "XKCD",
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/cpOto6I.jpg",
+  "thumbnail": "https://i.imgur.com/sdVsYvZ.png",
```
Comment (Alanexei) [inline websites/X/XKCD/dist/metadata.json:17]: Reviewers, check this if you haven't already.

**Review by Alanexei — APPROVED** [review]

**Review by Bas950 — CHANGES_REQUESTED** [review]
logo must be square like the one you use in discord

**Review by Bas950 — APPROVED** [review]

**Review by doomlerd — APPROVED** [review]

**Pull request merged**